pkg/minishift/clusterup/clusterup.go

```diff
@@ -58,10 +60,12 @@ type ClusterUpConfig struct {
 	OcPath           string
 	AddonEnv         []string
 	PublicHostname   string
+	SSHCommander     provision.SSHCommander
+	OcBinaryInsideVM string
 }
 
-// ClusterUp downloads and installs the oc binary in order to run 'cluster up'
-func ClusterUp(config *ClusterUpConfig, clusterUpParams map[string]string, runner util.Runner) error {
+// ClusterUp execute oc binary in order to run 'cluster up'
```

Comment:
I think this description needs to be updated with this function is returning.

Reply:
@budhrg this is I will do in an upcoming PR not in this atm.

Reply:
Ack @praveenkumar